insights/specs/default.py

```diff
@@ -605,11 +603,6 @@ def pcp_enabled(broker):
     redhat_release = simple_file("/etc/redhat-release")
     resolv_conf = simple_file("/etc/resolv.conf")
     rhn_conf = first_file(["/etc/rhn/rhn.conf", "/conf/rhn/rhn/rhn.conf"])
-    rhn_entitlement_cert_xml = first_of([glob_file("/etc/sysconfig/rhn/rhn-entitlement-cert.xml*"),
-                                   glob_file("/conf/rhn/sysconfig/rhn/rhn-entitlement-cert.xml*")])
-    rhn_schema_version = simple_command("/usr/bin/rhn-schema-version")
-    rhn_taskomatic_daemon_log = first_file(["/var/log/rhn/rhn_taskomatic_daemon.log",
-                                               "rhn-logs/rhn/rhn_taskomatic_daemon.log"])
```

Comment:
Please include all of these specs being removed from `default.py` except `rhn_schema_version` to `sos_archive.py` so that they will still be collected if present in sosreports.  If you can identify where `rhn_schema_version` should be located in a sosrport you should also add that to `sos_archive.py`.

Reply:
Done. rhn-schema-version won't be available in current sosreports as it was deprecated in https://github.com/sosreport/sos/commit/ad537d205ccc2b105b7362ab673fb90560d7725a but it still added where it used to be.